example/src/screens/Examples/AwareScrollView/index.tsx

```diff
@@ -75,7 +75,7 @@ export default function AwareScrollView({ navigation }: Props) {
         {new Array(10).fill(0).map((_, i) => (
           <TextInput
             key={i}
-            contextMenuHidden={i === 4}
+            contextMenuHidden={i === 4 && Platform.OS === "ios"}
```

Comment:
I had to use the hack with hidden context menu on iOS, because for some reasons on CI I'm getting different items comparing to my local simulator (I'm also getting `Share` and other items on CI)
On Android there is no such problem, so I decided to be a little bit more specific

Starting from 0.76 they disabled text selection if context menu is hidden, so you literally select text and after 16ms cursor gets moved to the end of text - and I don't handle the scenario where cursor is located a the end and i do not scroll

Because of that e2e test is broken - for now I just disabled context menu on iOS (because this was the original platform with the issue)

I hope in future versions of keyboard-controller I can re-work `KeyboardAwareScrollView` so that it will handle all cases (even when cursor is in the end).